Cargo.toml

```diff
@@ -147,7 +147,7 @@ version = "0.4.14"
 optional = true
 
 [dependencies.nix]
-version = "0.26.0"
+version = "0.26.2"
```

Comment:
Best to drop the change to Cargo.toml entirely. All the Cargo.lock should remain.